examples/sum/program/Cargo.toml

```
@@ -0,0 +1,8 @@
+[workspace]
+[package]
+version = "0.1.0"
+name = "sum-program"
+edition = "2021"
+
+[dependencies]
+sp1-zkvm = { git = "https://github.com/succinctlabs/sp1.git", rev = "v1.0.0-testnet"  }
```

This should be
```suggestion
sphinx-zkvm = { path = "../../../zkvm/entrypoint" }
```
instead